mapr/metadata.csv

```diff
@@ -0,0 +1,149 @@
+mapr.process.rss,gauge,,mebibyte,,The actual amount of memory in MB used by MapR processes.,0,mapr,
+mapr.process.data,gauge,,mebibyte,,The amount memory in MB used by the data segments of MapR processes.,0,mapr,
+mapr.process.cpu_percent,gauge,,percent,,The percentage of CPU used for MapR processes.,0,mapr,
+mapr.process.mem_percent,gauge,,percent,,The percentage of total system memory (not capped by MapR processes) used for MapR processes.,0,mapr,mem_percent
+mapr.process.cpu_time.syst,gauge,,second,,The amount of time measured in seconds that the process has been in kernel mode.,0,mapr,
+mapr.process.cpu_time.user,gauge,,second,,The amount of time measured in seconds that the process has been in user mode,0,mapr,
+mapr.process.disk_ops.read,gauge,,read,,The number of read operations for MapR processes.,0,mapr,
+mapr.process.disk_ops.write,gauge,,write,,The number of write operations for MapR processes.,0,mapr,
+mapr.process.disk_octets.read,gauge,,byte,,The number of bytes read from disk for MapR processes.,0,mapr,
+mapr.process.disk_octets.write,gauge,,byte,,The number of bytes written to disk for MapR processes.,0,mapr,
+mapr.process.page_faults.majflt,gauge,,error,,The number of major MapR process faults that required loading a memory page from disk.,0,mapr,
+mapr.process.page_faults.minflt,gauge,,error,,The number of minor MapR process faults that required loading a memory page from disk.,0,mapr,
+mapr.process.context_switch_voluntary,gauge,,process,,The number of voluntary context switches for MapR processes.,0,mapr,
+mapr.process.context_switch_involuntary,gauge,,operation,,The number of involuntary context switches for MapR processes.,0,mapr,
+mapr.streams.listen_msgs,gauge,,object,,The number of Streams messages read by the consumer.,0,mapr,
+mapr.streams.listen_rpcs,gauge,,object,,The number of Streams consumer RPCs.,0,mapr,
+mapr.streams.listen_bytes,gauge,,mebibyte,,The number of megabytes consumed by Streams messages.,0,mapr,
+mapr.streams.produce_msgs,gauge,,object,,The number of Streams messages produced.,0,mapr,
+mapr.streams.produce_rpcs,gauge,,object,,The number of Streams producer RPCs.,0,mapr,
+mapr.streams.produce_bytes,gauge,,mebibyte,,The number of megabytes produced by Streams messages.,0,mapr,
+mapr.streams.listen_currpcs,gauge,,object,,The number of concurrent Stream consumer RPCs.,0,mapr,
+mapr.rpc.bytes_read,gauge,,byte,,The number of bytes received by the MapR Filesystem over RPC.,0,mapr,
+mapr.rpc.bytes_sent,gauge,,byte,,The number of bytes sent by the MapR Filesystem over RPC.,0,mapr,
+mapr.rpc.calls_recd,gauge,,message,,The number of RPC calls received by the MapR Filesystem.,0,mapr,
+mapr.topology.utilization,gauge,,percent,,	The aggregate percentage of CPU utilization.,0,mapr,
+mapr.topology.disks_used_capacity,gauge,,gibibyte,,The amount disk space used in gigabytes.,0,mapr,
+mapr.topology.disks_total_capacity,gauge,,gibibyte,,The disk capacity in gigabytes. ,0,mapr,
```

Comment:
```suggestion
mapr.topology.disks_total_capacity,gauge,,gibibyte,,The disk capacity in gigabytes.,0,mapr,
```
gibibytes or gigabytes?

Reply:
MapR documentation says gigabytes, metadata validation only allows for `gibibyte` 🤷‍♀